templates/web-application/.storybook/main.ts

```diff
@@ -0,0 +1,55 @@
+import type { StorybookConfig } from "@storybook/react-webpack5";
+import type  { Configuration } from "webpack";
+
+const config: StorybookConfig = {
+  stories: [
+    "../src/**/*.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
With this template, is there a reason why someone would use a `.js` or `.jsx` files?

If I remember correctly, the other tools are only configured to handle `.ts` and `.tsx` files.